content/en/docs/migration/helm-migration.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Migrate to the Helm Controller"
+linkTitle: "Migrate from the Helm Operator"
+description: "How to migrate from Helm Operator to Flux v2 and its Helm controller."
+weight: 30
+card:
+  name: migration
+  weight: 20
+---
+
+## Prerequisites
+
+- Completed initial migration with the Flux migration guide
+
+## Migration strategy
+
+Due to the high number of changes to the API spec, there are no detailed instructions available to provide a simple migration path. But there is a [simple procedure to follow](#steps), which combined with the detailed list of [API spec changes](#api-spec-changes) should make the migration path relatively easy.
```

Comment:
There are 3 different methods for migrating resources across

- All at once
- Gradual with scaledown
- Gradual without scaledown

During migration, you need to change your manifests to be compatible with the new type, this involves
- defining a source `HelmRepository`, `GitRepository`, `Bucket`
- defining a `HelmRelease`
For more information about the changes see the api changes, and the guide on how to release helm